cfgov/cal/views.py

```diff
@@ -10,17 +10,19 @@
 from rest_framework import generics
 
 from django.shortcuts import render, render_to_response
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 from django import forms
 from django.utils.safestring import mark_safe
 from django.template import RequestContext
 from django.template.loader import get_template
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 from django.core.exceptions import ObjectDoesNotExist
 from django.db.models import Max, Min, Q
+from django.contrib import messages
 
 from cal.models import CFPBCalendar, CFPBCalendarEvent
-
+from v1.util.util import ERROR_MESSAGES
+from sheerlike.templates import get_date_string
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e